### DISCUSSION TITLE
Collecting blocked spam forms into the submissions archive

### DISCUSSION THREAD

**Discussion by Damman-Zhang** [elementor/elementor#27395] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I'm happy that the new elementor form can be used with Akisamt Anti-spam to block spam form submissions, but I have no way of knowing if any valid forms will be blocked by mistake, Akisamt Anti-spam blocks blog comments with a dedicated list of blocked spam comments in the wordpress admin panel, is it possible to do the same with the Elementor form?

### Describe the solution you'd like.

A new tab in the submissions archive is used to collect spam forms, but spam forms are not notified via email, which allows administrators to determine for themselves if there are valid forms in the spam archive

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by yonifre** [comment]
If you use Maspik, a powerful spam protection plugin that fully integrates Elementor forms and WordPress comments, you will find all blocked entries in the spam log. Additionally, Maspik provides more effective spam blocking than Akismet, ensuring fewer false positives and better overall protection.
